prime-router/docs/design/design/RS-Extention-Registry.yml

```diff
@@ -0,0 +1,5930 @@
+                          "extension": [
+                            {
+                              "url": "https://reportstream.cdc.gov/fhir/StructureDefinition/universal-id",
+                              "valueString": "2.16.840.1.113883.9.82"
+                            },
+                            {
+                              "url": "https://reportstream.cdc.gov/fhir/StructureDefinition/universal-id-type",
+                              "valueCode": "ISO"
+                            }
+                          ]
+                        }
+                      ],
+                      "value": "LAB_PRU_COMPONENT"
+                    }
+                  },
+                  {
+                    "url": "MSH.24",
+                    "extension": [
+                      {
+                        "url": "https://reportstream.cdc.gov/fhir/StructureDefinition/namespace-id",
+                        "valueString": "Alt Sending App"
+                      },
+                      {
+                        "url": "https://reportstream.cdc.gov/fhir/StructureDefinition/universal-id",
+                        "valueString": "Alt Sending App Universal"
+                      },
+                      {
+                        "url": "https://reportstream.cdc.gov/fhir/StructureDefinition/universal-id-type",
+                        "valueString": "UUID"
+                      }
+                    ]
+                  },
+                  {
+                    "url": "MSH.25",
+                    "extension": [
+                      {
+                        "url": "https://reportstream.cdc.gov/fhir/StructureDefinition/namespace-id",
+                        "valueString": "Sending App"
+                      },
+                      {
+                        "url": "https://reportstream.cdc.gov/fhir/StructureDefinition/universal-id",
+                        "valueString": "Sending App Universal"
+                      },
+                      {
+                        "url": "https://reportstream.cdc.gov/fhir/StructureDefinition/universal-id-type",
+                        "valueString": "ISO"
+                      }
+                    ]
+                  }
+                ]
+              }
+            ]
+          }
+        }
+      Extension:
+        - id: sender-id
+          url: "https://reportstream.cdc.gov/fhir/StructureDefinition/sender-id"
+          valueType: String
+          hl7Use: NA
+          description: Identifies the entity sending the bundle to ReportStream
+        - id: encoding-characters
+          url: "https://reportstream.cdc.gov/fhir/StructureDefinition/encoding-characters"
+          valueType: String
+          hl7Use: MSH.2
+        - id: character-set
+          url: "https://reportstream.cdc.gov/fhir/StructureDefinition/character-set"
+          valueType: String
+          hl7Use: MSH.18
+        - id: MSH.7
```

Comment:
Wondering if this might be a little confusing since this is technically nested in the msh-message-header extension?  Maybe we can follow the same pattern as MSH.24?

Reply:
Yup, that was a mistake. Fixed